darwin/future/core/items/delete_items.py

```diff
@@ -19,26 +19,28 @@ def delete_list_of_items(
     Parameters
     ----------
     client: Client
-        The client to use for the request
+        The client to use for the request.
     team_slug: str
-        The slug of the team containing the items
-    dataset_id: int
-        The ID of the dataset containing the items
-    item_ids: List[UUID]
-        The IDs of the items to be deleted
+        The slug of the team containing the items.
+    dataset_ids: (int | List[int])
```

Comment:
Same comment as before, this looks like it suggests a tuple
```suggestion
    dataset_ids: int | List[int]
```